pages/api/pets/index.js

```diff
@@ -0,0 +1,29 @@
+import dbConnect from "@/db/connect";
+import Pet from "@/db/models/Pet";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "GET") {
+    try {
+      const pets = await Pet.find();
+      response.status(200).json(pets);
+    } catch (error) {
+      console.error(error);
+      response.status(400).json({ error: error.message });
+    }
+  }
+
+  if (request.method === "POST") {
+    try {
+      const petData = request.body;
+      await Pet.create(petData);
+      // const pet = new Pet(petData);
+      // await pet.save();
+      response.status(201).json({ status: "Pet Created!" });
+    } catch (error) {
+      console.error(error);
+      response.status(400).json({ error: error.message });
+    }
+  }
```

Comment:
don't you also need PUT/PATCH to update?

Reply:
no because we never update the objects they will be like they are